server/app/models/user.rb

```diff
@@ -21,4 +24,17 @@ def name
     self.email
   end
 
+  ##
+  # @param [String] role
+  def in_role?(role)
+    self.roles.where(name: role).exists?
+  end
+
+  def master_admin?
+    self.in_role?('Master admin')
```

Comment:
My gut feeling says that pointing to these roles by names is not good idea. At least hide them behind some variables or constants for now.